lib/web/static/js/app.js

```diff
@@ -0,0 +1,81 @@
+const error_alert_markup = `
+<div class="alert alert-danger alert-dismissible fade show" role="alert">
+  Error on the request to get more resources of this state.
+  <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+    <span aria-hidden="true">&times;</span>
+  </button>
+</div>`
+
+class API {
+  constructor() { this.url = "/" }
+
+  getResourcesForState(list, callback) {
+    let xhr = new XMLHttpRequest();
+    let state = list.dataset.state
```

Comment:
Missing semicolon.

[Read more about it here.](http://eslint.org/docs/rules/semi)